neo/rawio/openephysbinaryrawio.py

```diff
@@ -0,0 +1,403 @@
+"""
+This module implement the "new" binary OpenEphys format.
+In this format channels are interleaved in one file.
+
+
+See
+https://open-ephys.github.io/gui-docs/User-Manual/Recording-data/Binary-format.html
+
+Author: Julia Sprenger and Samuel Garcia
+"""
+
+
+import os
+import re
+import json
+
+from pathlib import Path
+
+import numpy as np
+
+from .baserawio import (BaseRawIO, _signal_channel_dtype, _signal_stream_dtype,
+    _spike_channel_dtype, _event_channel_dtype)
+
+
+class OpenEphysBinaryRawIO(BaseRawIO):
+    """
+    Handle several Block and several Segment.
+
+
+    # Correspondencies
+    Neo          OpenEphys
+    block[n-1]   experiment[n]    New device start/stop
+    segment[s-1] recording[s]     New recording start/stop
+
+    handle several signals stream
+    handle events special npy data are respresented as array_annotations
+    do not handle spikes at the moment can be implemented if user demand
```

Comment:
```suggestion
    This IO handles several signal streams.
    Special event (npy) data are represented as array_annotations.
    The current implementation does not handle spiking data, this will be added upon user request
```